borg/archiver.py

```diff
@@ -804,6 +801,8 @@ def __call__(self, parser, ns, values, option):
                                help="""toggle progress display while creating the archive, showing Original,
                                Compressed and Deduplicated sizes, followed by the Number of files seen
                                and the path being processed, default: %(default)s""")
+        subparser.add_argument('--changed', action='store_true', dest='changed', default=False,
+                               help="""display which files were added to the archive""")
```

Comment:
ah, yes.
but: in fact, an archive has always all files. but it only stores new chunks, if it does not have them yet.